cypress/integration/conditions/boolean_conditions.cy.js

```diff
@@ -0,0 +1,168 @@
+import {
+    selectProgramDimensions,
+    INPUT_EVENT,
+    selectPeriod,
+    FIXED,
+    getPreviousYearStr,
+    getLineListTable,
+} from '../../helpers/index.js'
+import { EXTENDED_TIMEOUT } from '../../support/util.js'
+
+const dimensionName = 'MCH BCG dose'
+const periodLabel = 'Report date'
+
+const setUpTable = () => {
+    selectProgramDimensions({
+        inputType: INPUT_EVENT,
+        programName: 'Child Programme',
+        stageName: 'Birth',
+        dimensions: [dimensionName],
+    })
+
+    selectPeriod({
+        periodLabel,
+        category: FIXED,
+        period: {
+            type: 'Daily',
+            year: `${getPreviousYearStr()}`,
+            name: `${getPreviousYearStr()}-01-01`,
+        },
+    })
+
+    cy.getWithDataTest('{menubar}').contains('Update').click()
+
+    getLineListTable().find('tbody').should('be.visible')
+}
+
+const addConditions = (conditions) => {
+    cy.getWithDataTest('{layout-chip}').contains(dimensionName).click()
+    conditions.forEach((conditionName) => {
+        cy.getWithDataTest('{conditions-modal-content}')
+            .findWithDataTest('{dhis2-uicore-checkbox}')
+            .contains(conditionName)
+            .click()
+    })
+    cy.getWithDataTest('{conditions-modal}').contains('Update').click()
+}
+
+const assertTableMatchesExpectedRows = (expectedRows) => {
+    getLineListTable()
+        .find('tbody > tr')
+        .should('have.length', expectedRows.length)
+
+    expectedRows.forEach((val) => {
+        getLineListTable().find('tbody').contains('td', val)
+    })
+}
+
+const assertTooltipContainsEntries = (entries) => {
+    entries.forEach((entry) =>
+        cy.getWithDataTest('{tooltip-content}').contains(entry)
+    )
+}
+
+describe('boolean conditions', () => {
+    beforeEach(() => {
+        cy.visit('/', EXTENDED_TIMEOUT)
+        setUpTable()
+    })
+
+    it('Yes only', () => {
```

Comment:
Maybe we can rename this just to not confuse it with the `TRUE_ONLY` value type (my head just slipped on this for a few seconds 🙈 . e.g. `Yes selected` or `Only yes selected`...
```suggestion
    it('Yes selected', () => {
```